milatools/cli/commands.py

```diff
@@ -35,8 +36,24 @@ def main():
         exit(
             "Error: 'mila ...' should be run on your local machine and not on the Mila cluster"
         )
-    auto_cli(milatools)
-
+    
+    try:
+        auto_cli(milatools)
+    except Exception:
+        options = {
+            "labels": ",".join([sys.argv[1], mversion]),
+            "template": "bug_report.md",
+            "title": f"[v{mversion}] Issue running the command `mila "+
+                      "".join(sys.argv[1:])+"`"
```

Comment:
Might be a bad idea to log all args here in case some secrets are passed?

Reply:
It's unlikely secrets would be passed as command line arguments, but in any case, including all the arguments would make the title unwieldy. Maybe just mila + sys.argv[1]

Reply:
Done as suggested